addons/udes_get_info/models/stock_location.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+from odoo import models, fields, api, _
+
+
+class Location(models.Model):
+
+    _inherit = "stock.location"
+    _get_info_field_names = {
+        "id",
+        "name",
+        "display_name",
+        "complete_name",
+        "barcode",
+        "posx",
+        "posy",
+        "posz",
+        "scrap_location",
+        "return_location",
```

Comment:
Is storage format field ported ? I think should be good if storage format exist on stock.location to be by default in get_info

Reply:
Not yet in v14, agree it would be good to have here when it is. :)